packages/native/storybook/stories/Loader/ProgressLoader.stories.tsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
 import { storiesOf } from "../storiesOf";
-import { number } from "@storybook/addon-knobs";
+import { number, color, boolean } from "@storybook/addon-knobs";
 import { action } from "@storybook/addon-actions";
-import Loader from "../../../src/components/Loader/ProgressLoader";
-import { Icons } from "../../../src/assets";
+import { Icons, ProgressLoader } from "src";
```

Comment:
I think that a relative path is needed here otherwise storybook will fail to build.